src/linear_algebra/basic.lean

```diff
@@ -1330,11 +1330,14 @@ to_equiv_injective (equiv.eq_of_to_fun_eq h)
 section
 variable (M)
 
+variable (R)
```

Comment:
You could merge this line with the previous one, writing just `variables (M R)`.